### DISCUSSION TITLE
How to check the region where the project is deployed?

### DISCUSSION THREAD

**Discussion by IvanPaqmind** [vercel/vercel#4878] [Help] [answered]
I can't find any information about deployment regions in Vercel Admin.

```
curl -I my-site.com
```

displays the same region no matter what `regions: [__]` I set in `vercel.json`. 
I suspect this is because of CDN. *I'm in PRO trial if that matters*.

How to check the region where the project is deployed?

**Comment by styfle** [comment, marked as answer]
There's a header called [`x-vercel-id`](https://vercel.com/docs/v2/edge-network/headers) that contains the [region](https://vercel.com/docs/v2/edge-network/regions).

The [`regions`](https://vercel.com/docs/configuration#project/regions) property in `vercel.json` only applies to Serverless Functions. Static files are served from every region at the Edge.

**Comment by hyunjunian** [comment]
@styfle 
I am using the pro plan too.
I like vercel, but it's very inconvenient to not be able to check the distribution area.

**Comment by lirbank** [comment]
Hey, We're on the pro plan (paid, not the trial) and my `vercel.json` file looks like this:

```json
{
  "regions": ["iad1"]
}
```

But when I check the header for a serverless function it says I'm still on `sfo1`: `x-vercel-id: sfo1::...`

Anyone knows how to set the region properly?

**Comment by lirbank** [comment]
For anyone searching for this, you can check the AWS_REGION env to see which region your function is in. 

https://vercel.com/docs/platform/limits#reserved-variables

**Comment by 10xchs** [comment]
I still don't understand how I can see the region. What's x-vercel-id and how do I read it? `process.env.AWS_REGION` is undefined.

**Reply by bigsee** [reply]
If you open the `Developer Tools` in your browser and look at the `Network` tab, you will see the requests being made. You can click on the request you are interested in e.g. `/` and then look at the `Headers` > `Response headers` to see something like this:

<img width="466" alt="Screenshot 2023-09-04 at 08 59 38" src="https://github.com/vercel/vercel/assets/11768967/aea962c4-45d8-4c90-b448-6c366e182210">

I guess the first four characters at the start signifies the zone, in my case `fra1` for Frankfurt.
